templates/devtest/gitea-ui.tmpl

```diff
@@ -212,18 +205,39 @@
 
 		<div>
 			<div class="ui dropdown mini button">
-				<span class="text">small dropdown</span>
+				<span class="text">mini dropdown</span>
 				{{svg "octicon-triangle-down" 12 "dropdown icon"}}
-				<div class="menu">
-					<div class="item">item</div>
-				</div>
+			</div>
+			<div class="ui dropdown tiny button">
+				<span class="text">tiny dropdown</span>
+				{{svg "octicon-triangle-down" 12 "dropdown icon"}}
+			</div>
+			<div class="ui button dropdown">
+				<span class="text">button dropdown</span>
+				{{svg "octicon-triangle-down" 14 "dropdown icon"}}
 			</div>
 			<div class="ui dropdown large button">
 				<span class="text">large dropdown</span>
 				{{svg "octicon-triangle-down" 18 "dropdown icon"}}
-				<div class="menu">
-					<div class="item">item</div>
-				</div>
+			</div>
+		</div>
+
+		<div>
+			<div class="ui dropdown mini compact button">
+				<span class="text">mini compact</span>
+				{{svg "octicon-triangle-down" 12 "dropdown icon"}}
+			</div>
+			<div class="ui dropdown tiny compact button">
+				<span class="text">tiny compact</span>
+				{{svg "octicon-triangle-down" 12 "dropdown icon"}}
+			</div>
+			<div class="ui button compact dropdown">
+				<span class="text">button compact</span>
+				{{svg "octicon-triangle-down" 14 "dropdown icon"}}
+			</div>
+			<div class="ui dropdown large compact button">
+				<span class="text">large compact</span>
+				{{svg "octicon-triangle-down" 18 "dropdown icon"}}
```

Comment:
Hmm, do we really need to alter the SVG size? I was under the impression 14px works for all button sizes.

Reply:
Not a must. For example, the "Operations" button still uses "size 14", which is also fine.